### ISSUE TITLE
Add accessibility information to the website

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1025]
I was looking into a shop I'd like to go to and found accessibility info in a google review, but not on the website. Went looking at the DoES website and on my phone, I can't find anything. We mention bike storage (good!) but not accessibility - which is silly, because we're very decent in that respect, and willing to work to make things better.

I can't remember how much I put on the wiki, and I can't seem to access it just now (suspected user error) but I'm willing to write the content, maybe run it by @cheapjack in case I'm missing anything.

Just because often, if that information is hard to find, I'll just not go rather than contact them/most often get a limp response that it's impossible for some reason or another. 

...Well that was a weird time to come across the "invalid" label and completely misinterpret it!

**Comment by johnmckerrell** [comment]
Just assigned to @willcrobinson hopefully you can add the accessible icon (http://accessibleicon.org) and some text to the front page, and is it possible for those sections to link to another page? The wiki page on accessibility would seem to make sense[1]. I'll suggest the following text but perhaps @Robotorium you might have a better idea:

ACCESSIBLE SPACE
We want everyone to feel welcome and have tried our best to make that possible.


1. https://github.com/DoESLiverpool/somebody-should/wiki/Accessibility

**Comment by willcrobinson** [comment]
Create accessibility page (https://doesliverpool.com/accessibility/) and have linked to it from the front page.

I've also added a link to the footer of the website.

**Comment by Robotorium** [comment]
Looks brilliant.

Op wo 9 jan. 2019 om 11:57 schreef Will Robinson <notifications@github.com>:
